keyboards/xd75/keymaps/bulbizarre/readme.md

```diff
@@ -0,0 +1,4 @@
+# Bulbizarre layout
```

Comment:
```suggestion
# Bulbizarre keymap
```

This is actually a keymap, in QMK layout pertains more to the physical layout and is characterized by the LAYOUT macro.